demo/vl-upload.html

```diff
@@ -25,84 +29,84 @@ <h1>vl-ui-upload demo</h1>
+            &lt;vl-upload url="http://httpbin.org/post" error&gt;&lt;/vl-upload&gt;
         </code>
       </pre>
       <div class="demo">
-        <vl-upload url="http://example.com" input-name="files" error></vl-upload>
+        <vl-upload id="vl-upload-error" url="http://httpbin.org/post" input-name="files" error></vl-upload>
       </div>
     </div>
 
     <div>
       <h2>Upload Clear</h2>
       <pre>
         <code class="line-numbers language-markup">
-            &lt;vl-upload url="http://example.com"&gt;&lt;/vl-upload&gt;
+            &lt;vl-upload url="http://httpbin.org/post"&gt;&lt;/vl-upload&gt;
 
             &lt;button is="vl-button" onclick="clear()"&gt;Clear&lt;/button&gt;
 
             &lt;script type="text/javascript"&gt;
               function clear() {
-                document.querySelector("vl-upload[id=\"upload-4\"]").clear();
+                document.querySelector("#vl-upload-clear").clear();
               }
             &lt;/script&gt;
         </code>
       </pre>
       <div class="demo">
-        <vl-upload id="upload-4" url="http://example.com" input-name="files"></vl-upload>
+        <vl-upload id="vl-upload-clear" url="http://httpbin.org/post" input-name="files"></vl-upload>
 
-        <button is="vl-button" onclick="clearList()">Clear</button>
+        <button id="vl-upload-clear-button" is="vl-button" onclick="clearList()">Clear</button>
 
         <script type="text/javascript">
           function clearList() {
-            document.querySelector("vl-upload[id=\"upload-4\"]").clear();
+            document.querySelector("#vl-upload-clear").clear();
           }
         </script>
       </div>
     </div>
 
     <div>
       <h2>Beperkte Upload (aantal en grootte)</h2>
-      Beperkt aantal bestanden opladen, met een bepaalde grootte (20MB).
+      Beperkt aantal bestanden opladen, met een bepaalde grootte (2MB).
       <pre>
         <code class="line-numbers language-markup">
-            &lt;vl-upload url="http://example.com"
+            &lt;vl-upload url="http://httpbin.org/post"
               input-name="files"
               error-message-filesize="De grootte van het bestand mag maximaal :maxFsz zijn."
               error-message-maxfiles="Je kan maximaal :maxfl file(s) uploaden."
-              max-files="5" max-size="20000000"&gt;
+              max-files="5" max-size="2000000"&gt;
             &lt;/vl-upload&gt;
         </code>
       </pre>
       <div class="demo">
-        <vl-upload url="http://example.com" input-name="files"
+        <vl-upload id="vl-upload-max-5" url="http://httpbin.org/post" input-name="files"
                    error-message-filesize="De grootte van het bestand mag maximaal :maxFsz zijn."
                    error-message-maxfiles="Je kan maximaal :maxfl file(s) uploaden." max-files="5"
-                   max-size="20000000"></vl-upload>
+                   max-size="2000000"></vl-upload>
```

Comment:
Waarom was dit nodig?

Reply:
anders moest ik ne file van meer dan 20MB mee in de Github steken. alternatief was om programmatorisch nog ne keer aan te passen.